sound/soc/sof/amd/acp.c

```diff
@@ -322,10 +369,22 @@ static int acp_init(struct snd_sof_dev *sdev)
 	return acp_reset(sdev);
 }
 
+static const struct sof_amd_acp_desc
+	*get_chip_info(struct snd_sof_pdata *pdata)
```

Comment:
I'm not sure this is the right way to indent functions where the return type is on a separate line...